src/devices/RadioTransmitter/Devices/Kt0803/BassBoost.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.RadioTransmitter
+{
+    /// <summary>
+    /// Bass Boost
+    /// </summary>
+    public enum BassBoost : byte
+    {
+        /// <summary>
+        /// Disable
+        /// </summary>
+        Boost_Disable = 0,
```

Comment:
please remove underscores since lack of them doesn't impede legibility of the code